logstash-core/src/main/java/org/logstash/config/ir/graph/Graph.java

```diff
@@ -222,7 +223,6 @@ private Graph chain(Collection<Vertex> fromLeaves, Collection<Vertex> toVertices
 
     public Collection<Edge> chainVertices(Edge.EdgeFactory edgeFactory, Vertex... argVertices) throws InvalidIRException {
         Collection<Edge> edges = chainVerticesUnsafe(edgeFactory, argVertices);
-        validate();
```

Comment:
We always validate the PipelineIR object, this is a nice invariant, but now that we know speed is a concern, just not worth it.